### DISCUSSION TITLE
apk v3.8.2 not parsing package for install

### DISCUSSION THREAD

**Discussion by Djtrip83** [KRTirtho/spotube#1953] [Q&A]
### Is there an existing issue for this? (Please read the description)

- [X] I have searched the existing issues

### Current Behavior

I tried both update and clean install - the APK of v3.8.2 is not installing. I tried another release (v3.8.0) and the APK installation works 

### Expected Behavior

Apk should update old version of app and install

### Logs

N/a

### Operating System

android pixel 6a

### Installation source

Website (spotube.krtirtho.dev), GitHub Releases (Binary)

### Self grab

- [ ] I'm ready to work on this issue!

**Comment by Bowarc** [comment]
I have the same issue.

Btw there is no release for 3.8.2 on F-Droid.

**Comment by KRTirtho** [comment]
uninstall the old version. Then install.

**Reply by Djtrip83** [reply]
Did that already twice. The issue persists. No remnants found from uninstall. What else can it be?

**Reply by Djtrip83** [reply]
Did that already twice. The issue persists. No remnants found from uninstall. What else can it be?

**Reply by KRTirtho** [reply]
Can you try installing from playstore?

**Reply by Djtrip83** [reply]
It works when installing from PlayStore!
Resolved
